package-lock.json

```diff
@@ -3025,6 +3025,21 @@
         "@fortawesome/fontawesome-common-types": "^0.2.32"
       }
     },
+    "@fortawesome/free-brands-svg-icons": {
+      "version": "5.15.3",
+      "resolved": "https://registry.npmjs.org/@fortawesome/free-brands-svg-icons/-/free-brands-svg-icons-5.15.3.tgz",
+      "integrity": "sha512-1hirPcbjj72ZJtFvdnXGPbAbpn3Ox6mH3g5STbANFp3vGSiE5u5ingAKV06mK6ZVqNYxUPlh4DlTnaIvLtF2kw==",
+      "requires": {
+        "@fortawesome/fontawesome-common-types": "^0.2.35"
+      },
+      "dependencies": {
+        "@fortawesome/fontawesome-common-types": {
+          "version": "0.2.35",
+          "resolved": "https://registry.npmjs.org/@fortawesome/fontawesome-common-types/-/fontawesome-common-types-0.2.35.tgz",
+          "integrity": "sha512-IHUfxSEDS9dDGqYwIW7wTN6tn/O8E0n5PcAHz9cAaBoZw6UpG20IG/YM3NNLaGPwPqgjBAFjIURzqoQs3rrtuw=="
+        }
+      }
+    },
```

Comment:
Looks like we want to use a different version of this dependency so that we aren't changing the  `@fortawesome/fontawesome-common-types` version.

This test should have failed based on this I believe..

- https://github.com/sparkletown/sparkle/blob/staging/package-lock.test.ts#L16-L40

---

Looking at the versions, I believe we want `5.15.1` here:

- https://github.com/FortAwesome/Font-Awesome/blob/5.15.1/js-packages/@fortawesome/free-brands-svg-icons/package.json#L49-L51

Reply:
> This test should have failed based on this I believe..
> 
> - https://github.com/sparkletown/sparkle/blob/staging/package-lock.test.ts#L16-L40

Bonus points if you figure out why that test wasn't failing here and fix up the bug in it.

Reply:
@oleksii-front This still seems to be outstanding?

Reply:
@oleksii-front This still seems to be outstanding?

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1396/commits/a611e3516ee9464e51516aa5c868075acd54f997